### PR TITLE
mgmt, draft generate fluent lite for resource RP

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
root pom.xml and ci.yml; README and CHANGELOG is manual.

generated by https://github.com/Azure/autorest.java/pull/738

### CODE REVIEW

File: sdk/resources/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+pr:
+  branches:
+    include:
+      - master
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+        - sdk/reosurces/
```
Comment (weshaggard) [inline sdk/resources/ci.yml:22]:
```suggestion
        - sdk/resources/
```

File: sdk/resources/ci.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+trigger:
+  branches:
+    include:
+      - master
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/reosurces/
```
Comment (weshaggard) [inline sdk/resources/ci.yml:11]:
```suggestion
      - sdk/resources/
```

File: sdk/resources/azure-resourcemanager-resource/pom.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        </developer>
+    </developers>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core-management</artifactId>
```
Comment (weidongxu-microsoft) [inline sdk/resources/azure-resourcemanager-resource/pom.xml:46]: code generator will be improved to output `<!-- {x-version-update;com.azure:azure-core-management;dependency} -->` as well.

**Comment by weidongxu-microsoft** [comment]
no longer needed, as automation is almost done.

**Pull request closed without merging**